airbyte-integrations/connectors/source-twilio-singer/.dockerignore

```
@@ -0,0 +1 @@
+build
```

to be safe when publishing images
```suggestion
build
secrets
```